SDK v2 migration guide.md

```diff
@@ -41,6 +41,7 @@ to migrate to version 2.x when they have the chance. For more details on LTS rel
 | `ReceiveAsync` | `ReceiveMessageAsync` |
 | `CompleteAsync` | `CompleteMessageAsync` |
 | `SetConnectionStatusChangesHandler` | `SetConnectionStatusChangeHandler` |
+| `MessageResponse` | `MessageAcknowledgementType` |
```

Comment:
Do you think we need the word Type?

Reply:
I like it here, yeah. It fits for an enum.